src/main/java/org/minions/devfund/dennis/Actor.java

```diff
@@ -0,0 +1,126 @@
+package org.minions.devfund.dennis;
+
+import java.util.ArrayList;
+
+/**
+ * This is the methods for Actor class.
+ */
+public class Actor implements Comparable<Actor> {
+    private String name;
+    private ArrayList<Movie> movies;
+
+    /**
+     * Constructor for Actor.
+     * @param name of actor
+     */
+    public Actor(final String name) {
+        this.name = name;
+        movies = new ArrayList<>();
+    }
+
+    /**
+     * Default values for Constructor.
+     */
+    public Actor() {
+        this("");
+    }
+
+    /**
+     * Get name of actor.
+     * @return get the name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * set the name in the list.
+     * @param name of actor
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * get all movies of actor.
+     * @return arrayList of the movies
+     */
+    public ArrayList<Movie> getMovies() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)